<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonLoginWindow</class>
 <widget class="QMainWindow" name="CommonLoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="UNameLabel">
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="UNameLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="PwordLabel">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="PwordLineEdit"/>
         </item>
        </layout>
    </item>
    <item>
       <widget class="QPushButton" name="cancelPushButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
    </item>
    <item>
       <widget class="QPushButton" name="submitPushButton">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
